test/test_utils/sagemaker.py

```diff
@@ -224,7 +245,7 @@ def install_sm_local_dependencies(framework, job_type, image, ec2_conn, ec2_inst
     python_invoker = get_python_invoker(ec2_instance_ami)
     # Install custom packages which need to be latest version"
     # using virtualenv to avoid package conflicts with the current packages
-    ec2_conn.run(f"sudo apt-get install virtualenv -y ")
+    ec2_conn.run(f"sudo apt-get install virtualenv python3-pip -y ")
```

Comment:
Please check that `pip` doesn't already exist before installing system pip, because it will conflict with version that might already be installed, and cause test errors on non Graviton AMIs.

Reply:
it doesn't as the process was failing with "no pip found", being the reason for the add.